ehitchhiking-ui/src/app/shared/components/trip/trip.component.sass

```diff
@@ -2,8 +2,11 @@
 @import '../../../../styles/colors'
 @import '../../../../styles/constants'
 
-.position-car
+.position-role
   margin-left: $default-offset*2
+.truncated
```

Comment:
you should not override truncated, pls add other custom class

Reply:
changed truncated<->start-end-point